<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameEndWindow</class>
 <widget class="QMainWindow" name="GameEndWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(27, 27, 27)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="WonLossLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>891</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stencil</family>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray</string>
    </property>
    <property name="text">
     <string>Win!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayAgain">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stencil</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 20, 20);
color: gray;
border-style: outset;
border-width: 2px;
border-radius:15px;
border-color: rgb(20, 20, 20);</string>
    </property>
    <property name="text">
     <string>Play again</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EndMainMenu">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stencil</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 20, 20);
color: gray;
border-style: outset;
border-width: 2px;
border-radius:15px;
border-color: rgb(20, 20, 20);</string>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
